server/createDatabase.js

```diff
@@ -0,0 +1,68 @@
+const pgtools = require('pgtools');
+const { Pool } = require('pg');
+const generateCsv = require('./postgresDataSeed');
+const seedPostgres = require('./csvSeed.js');
+
+
+const config = {
+  user: 'postgres',
+  host: 'localhost',
+  password: 'samaung1',
+  port: 5432,
+};
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'games22',
+  password: 'samaung1',
+  port: 5432,
+});

```

Comment:
Are both of these necessary?  Also, I'd recommend adding your password into as an environment variable instead of committing it here on github.

Reply:
they are both necessary, but good point I should use a variable.